scripts/core/EXP-USM-IMPORT-RELEASE.rst

```diff
@@ -0,0 +1,58 @@
+
+<%
+    OneApi=tags['$OneApi']
+    x=tags['$x']
+    X=x.upper()
+%>
+.. _experimental-usm-import-release:
+
+==================
+USM Import/Release
+==================
+
+.. warning::
+
+    Experimental features:
+
+    *   May be replaced, updated, or removed at any time.
+    *   Do not require maintaining API/ABI stability of their own additions over
+        time.
+    *   Do not require conformance testing of their own additions.
+
+
+Data transfer between Host and Device is most efficient when source and
+destination are both allocated in USM memory.
+In situations where host data will participate in host/device transfers
+and the host data allocation is under user control, USM functions
+such as malloc_host could be used to allocate USM memory instead of
+system memory.
+However, this is not always possible if the source code where the allocation
+is made is not available, or source code changes are prohibited for portability
+reasons.
+In these situations a mechanism to temporarily promote system memory to USM
+for the duration of the host/device data transfers is useful for maximizing
+data transfer rate.
+
+
+Import Host Memory into USM
+===========================
+
+Import a range of host memory into USM.
+
+.. parsed-literal::
+
+    // Import into USM
+    USMImport(hContext, hostPtr, size);

```

Comment:
This fix works for me locally:

```suggestion
    ${x}USMImportExp(hContext, hostPtr, size);
```

Reply:
Making the change you suggest still causes this error:
Generating ../docs/source/core/EXP-USM-IMPORT-RELEASE.rst...
/iusers/rdeodhar/cmplr/ur_l0import1/llvm/scripts/core/EXP-USM-IMPORT-RELEASE.rst(45) : error : symbol '$xUSMImport' not found
/iusers/rdeodhar/cmplr/ur_l0import1/llvm/scripts/core/EXP-USM-IMPORT-RELEASE.rst(57) : error : symbol '$xUSMRelease' not found
Failed to generate specification.

Reply:
I don't think you've actually applied the suggested fix because those names are missing the `Exp` suffix.

Reply:
Yes, indeed. I have changed the name as you suggest now.
I then ran just the command "make -C build generate". That didn't change any of the generated files. Is that expected?